ts/features/fims/history/store/actions/index.ts

```diff
@@ -2,6 +2,7 @@ import { ActionType, createAsyncAction } from "typesafe-actions";
 import { ConsentsResponseDTO } from "../../../../../../definitions/fims/ConsentsResponseDTO";
 
 export type FimsHistoryGetPayloadType = {
+  isFirstRequest: boolean;
```

Comment:
no `continuationToken` != should clear the whole history, even though it could do it.

To be honest I prefer keeping it explicit, and for that matter I am also renaming the flag; LMK what you think!